patches/chrome-browser-ui-webui-theme_source.cc.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/chrome/browser/ui/webui/theme_source.cc b/chrome/browser/ui/webui/theme_source.cc
+index d8a4620d0525c6665ee783ac4a80470cc20e26c8..ee3802aa65ef011d717eb54269cdecd35a956610 100644
+--- a/chrome/browser/ui/webui/theme_source.cc
++++ b/chrome/browser/ui/webui/theme_source.cc
+@@ -325,11 +325,13 @@ void ThemeSource::SendColorsCss(
+   const auto* theme_service =
+       ThemeServiceFactory::GetForProfile(profile_->GetOriginalProfile());
+   std::string theme_id;
++  if (theme_service) {
```

Comment:
This seems strange, is there a reason why this would return `nullptr` for us? If it only is null in some certain known scenarios for us, perhaps we could just return a stub implementation that does nothing in those situations?